deploy/README.md

```diff
@@ -0,0 +1,309 @@
+
+If you did not change `docker-compose.yml`, you can update simply using `git pull`.
+
+```sh
+# TODO check BRANCH/TAG before release
+git pull
+```
+If you have made complex changes to `docker-compose.yml`, `git commit` them to a separate branch and `git merge` them with upstream changes as you see fit.
+
+> 💡 One major version of WBS Deployment Kit will always reference the only one major version for each of the WBS Service Images. Therefor, updating WBS Deployment Kit minor and patch versions from its major versions git branch will never lead to breaking changes in WBS Service Images. Those updates are always considered safe!
+
+#### Major upgrades
+
+Major version upgrades are performed by updating WBS Deployment Kits major version. This might reference new major versions of WBS Service Images too and therefor include breaking changes. Those may require additional steps. Refer to specific upgrade instructions in the [Migration Guide](#migration-guide) section below.
+
+WBS only supports updating from one major version to the next one. If you want to upgrade from 21 to 23, you need to upgrade from 21 to 22 first and in a next step from 22 to 23.
+
+> ‼️ Always [create a backup](#backup-your-data) of your data before performing a major version update.
+
+Once all version specific migration activities are done, use `git` to update your WBS Deployment Kit to the new version.
+
+```sh
+docker compose down 
+git remote update
+git checkout NEWVERSIONBRANCH
+git pull
+docker compose up --wait
+```
+
+#### Automatic updates
+At the moment, WBS Deployment Kit does not support automatic updates. To deploy minor and patch updates, including security fixes, [restart you instance](#minor-and-patch-updates-for-wbs-service-containers) on a regular basis, e.g. with a systemd timer or cron job.
+
+#### Downgrades
+Downgrades are not supported. In order to revert an update, restore your data from a backup made prior to the upgrade.
+
+### Removing Wikibase Suite Completely with all its Data
+
+‼️ **This will destroy all data, make sure to [backup anything](#backup-your-data) you wish to retain.**
+
+To reset the configuration and data, remove the Docker Containers, Docker Volumes and the generated `LocalSettings.php` file.
+
+```sh
+docker compose down --volumes
+rm config/LocalSettings.php
+```
+
+Removing the `traefik-letsencrypt-data` volume will request a new certificate from LetsEncrypt on next launch of your instance. Certificate generation on LetsEncrypt is [rate limited](https://letsencrypt.org/docs/rate-limits/). Eventually, you might be blocked from generating new certificates **for a couple of days**. To prevent that, switch to the LetsEncrypt staging server by appending the following to your `traefik` `command` in  your `docker-compose.yml` file:
```

Comment:
As I think there is still a comment to this effect in the `docker-compose.yml`. I would slightly prefer it only being documented in one place. Perhaps we make this warning here and point to the `docker-compose.yml` for further instructions? Alternatively, maybe we can leave this as-is and remove the commented-out section in `docker-compose.yml` with a brief warning there pointing back to the README docs in its place. ?